modules/site-health/admin-site-health.php

```diff
@@ -308,9 +310,17 @@ public function info_warning( $debug_info ) {
 			$fields['term-no-lang']['value'] = $this->format_array( $terms_no_lang );
 		}
 
+		// add WPML files
+		$wpml_files = PLL_WPML_Config::instance()->get_files();
+		if ( ! empty( $wpml_files ) ) {
+			$fields['wpml']['label'] = 'wpml-config.xml files';
+			$fields['wpml']['value'] = $wpml_files;
+		}
+
+		// Create the section
 		if ( ! empty( $fields ) ) {
 			$debug_info['pll_warnings'] = array(
-				'label'  => sprintf( esc_html__( 'Polylang warnings', 'polylang' ), POLYLANG ),
+				'label'  => sprintf( esc_html__( 'Polylang informations', 'polylang' ), POLYLANG ),
```

Comment:
```suggestion
				/* translators: placeholder is the plugin name */
				'label'  => sprintf( esc_html__( '%s informations', 'polylang' ), POLYLANG ),
```

Reply:
I had previously missed that `sprintf()` was used without placeholder.

Reply:
I've added a Placeholder and the comment